docs/ru/getting_started/example_datasets/ontime.md

```diff
@@ -193,7 +193,7 @@ SELECT Carrier, count(*) FROM ontime WHERE DepDelay>10 AND Year = 2007 GROUP BY
 Q5. Процент задержек по перевозчикам за 2007 год
 
 ``` sql
-SELECT Carrier, c, c2, c*1000/c2 as c3
+SELECT Carrier, c, c2, c*100/c2 as c3
```

Comment:
let's make it consistent: we either have spaces around math operators or we don't

Reply:
I've deleted spaces around multiplication sign in the file, but I don't know how to be consistent in this question.
Compare `x = a * y + b` and `x = a*y + b`. I'd prefer the second variant, but it is not consistent.